engine/lib/comments.php

```diff
@@ -31,23 +31,23 @@ function _elgg_comments_init() {
 /**
  * Page handler for generic comments manipulation.
  *
- * @param array $page
+ * @param array $segments
  * @return bool
  * @access private
  */
-function _elgg_comments_page_handler($page) {
+function _elgg_comments_page_handler($segments) {
 
-	$page = elgg_extract(0, $page);
```

Comment:
Easier to see in split view. This line used to redefine $page, now it creates a new var $page so we still have access to URL segments